experiments/reacher_ioc/hyperparams.py

```diff
@@ -109,17 +141,22 @@
             END_EFFECTOR_POINTS, END_EFFECTOR_POINT_VELOCITIES],
     'meta_include': [],
     'camera_pos': np.array([0., 0., 3., 0., 0., 0.]),
+    #'camera_pos': np.array([0., 0., 2., 0., 0.2, 0.5]),
     'target_end_effector': [np.concatenate([np.array([.1, -.1, .01])+ demo_pos_body_offset[i], np.array([0., 0., 0.])])
                             for i in xrange(DEMO_CONDITIONS)],
     'success_upper_bound': 0.01,
+    'render': True,
 }
 
 
 algorithm = {
     'type': AlgorithmTrajOpt,
     'ioc' : 'ICML',
-    'max_ent_traj': 1.0,
+    'global_cost': True,
+    'max_ent_traj': 0.001,
```

Comment:
be careful with this... (make sure max_ent_traj is set to 1 when you use IOC and/or fine-tuning)